CodyJones4235/PoopMap
# comments & Posts

#### Changes Made
1. Added file Createpost,commentlist,commentprovider,createcomment to posts,comment directory.
1. Modified file `Createpost &createcomment  ` to include `createpost & createcomment` and functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout cwj-still-posts
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Example: When user creates post/comment, then post/comment should appear.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
